spec/mock_data_spec.rb

```diff
@@ -8,10 +8,10 @@
         id: 10,
         name: 'Example 1'
       },
-      {
-        id: 11,
-        name: 'Example 2'
-      }]
+                {
+                  id: 11,
+                  name: 'Example 2'
+                }]
```

Comment:
What's going on here? Looks off to me.

Reply:
huh, not sure, this must be a rubocop thing? I didn't manually go to change anything in mock_data_spec.